adapters/repos/db/shard_lazyloader.go

```diff
@@ -86,7 +86,10 @@ func (l *LazyLoadShard) mustLoad() {
 
 func (l *LazyLoadShard) mustLoadCtx(ctx context.Context) {
 	if err := l.Load(ctx); err != nil {
-		panic(err.Error())
+		// only panic if multi-tenancy is not enabled, so that we don't take out the whole server
+		if !(l.shardOpts.class.MultiTenancyConfig != nil && l.shardOpts.class.MultiTenancyConfig.Enabled) {
```

Comment:
I may be better to modify `mustLoadCtx` to return an error
because if the shard was not successfully loaded but nothing happens then the actions will be attempted anyway without notice.